src/validation/validator.js

```diff
@@ -0,0 +1,19 @@
+function validate({
```

Comment:
no tests for the validate ? 😱 
😄

Reply:
i think its the most tested module in this repo :))